profiling/src/profiling.rs

```diff
@@ -922,4 +808,90 @@ impl Profiler {
             }
         }
     }
+
+    unsafe fn prepare_sample_message(
+        &self,
+        frames: Vec<ZendFrame>,
+        sample: i64,
+        wall_time: i64,
+        cpu_time: Option<i64>,
+        alloc_samples: i64,
+        alloc_size: i64,
```

Comment:
@morrisonlevi I was thinking of creating a struct for this, that could also make initialisation easier (default should always be `0`). Something like this
```rust
#[derive(Default)]
struct SampleValues {
    interrupt_count: i64,
    wall_time: i64,
    cpu_time: i64,
    alloc_samples: i64,
    alloc_size: i64
}

// usage:
let samples = SampleValues {
    alloc_samples: 1,
    alloc_size: 1024,
    ..Default::default()
};
```

WDYT?

Reply:
If we choose to not do it now, let's do this or something like it soon after merge. Clippy doesn't like how many arguments this function has :)